src/pages/Orgs/pages/OrgsId/pages/OrgRoot/components/LinkForm.tsx

```diff
@@ -0,0 +1,72 @@
+import { InputAdornment } from '@mui/material'
+import { ChangeEvent, useCallback, useEffect, useMemo, useState } from 'react'
+import { useSearchParams } from 'react-router-dom'
+
+import { UiButton, UiSearchField } from '@/ui'
+
+interface Props {
+  isLoading: boolean
+  onLinkIdChange: (linkId: string) => void
+}
+
+export default function LinkForm({ isLoading, onLinkIdChange }: Props) {
+  const [params] = useSearchParams()
+  const [linkOrLinkId, setLinkOrLinkId] = useState(params.get('linkId') ?? '')
+  const [isVerifying, setIsVerifying] = useState(false)
+
+  const linkId = useMemo(() => {
+    if (!isVerifying) return ''
+
+    try {
+      const url = new URL(linkOrLinkId)
+      return url.pathname.split('/').pop() ?? ''
```

Comment:
not sure if here will be undefined

<img width="169" alt="image" src="https://github.com/rarimo/rarime-app/assets/39623143/3e026168-be5b-4abc-9156-f5075b7b6105">

mb ```||``` ?